Backend/index.js

```diff
@@ -129,9 +142,16 @@ app.get("/products/:id", async function (req, res) {
     console.log("No barcode provided");
   } else if (req.params.id) {
     try {
+      let id = req.params.id;
+      let len = id.length;
+      // pads the incoming id with zeros
```

Comment:
I think we can utilize the padStart function

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
change implemented